gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts

```diff
@@ -153,23 +155,34 @@ export const wrapCommand = (command) => {
 
     executeProposal: ProposalAction = async (proposal: PublicKey, context): Promise<string> => {
       logger.loading(`Executing proposal`)
-
-      const tx = await this.program.rpc.executeTransaction({
-        accounts: {
-          multisig: this.multisigAddress,
-          multisigSigner: context.multisigSigner,
-          transaction: proposal,
-        },
-        remainingAccounts: context.proposalState.accounts
-          .map((t) => (t.pubkey.equals(context.multisigSigner) ? { ...t, isSigner: false } : t))
-          .concat({
-            pubkey: context.proposalState.programId,
-            isWritable: false,
-            isSigner: false,
-          }),
-      })
-      logger.info(`Execution TX hash: ${tx.toString()}`)
-      return tx
+      // get the command's starting word to map it to the respective IDL(ocr2, store etc)
+      const { idl } = getContract(command.id.split(':')[0], '')
```

Comment:
This is not good. This could be addressed when we change this. 
For the moment, leave it as it was
>  For the moment let's load the contract and program here too. Open an issue please to circle back here. You can use program.idl then

Reply:
The issue here if I remove this, is that errors will not be translated properly for multisig commands, leading to confusion for users. 

Maybe open an issue for this as well? Tackling it along with the other by keeping some program/idl state on SolanaCommand could be possible.

Reply:
Yes, open an issue please. I'd say that the same issue can solve both problems